uspace/srv/hid/display/test/window.c

```diff
@@ -1108,13 +1108,18 @@ PCUT_TEST(window_set_cursor)
 	PCUT_ASSERT_ERRNO_VAL(EINVAL, rc);
 	PCUT_ASSERT_EQUALS(wnd->display->cursor[dcurs_arrow], wnd->cursor);
 
+	// Check that invalid cursors cannot be set: ignore enum conversions
+	// as we are out-of-bounds
+#pragma GCC diagnostic push
+#pragma GCC diagnostic ignored "-Wenum-conversion"
```

Comment:
I believe there are more cases when we are using `enum` instead of macros. Perhaps in another PR?

BTW, it seems to me that there was one another comment from you but I cannot find it at the moment?